app/locales/en.json

```
@@ -183,6 +183,10 @@
     "epidemiologic_report_tab": "Epidemiological report",
     "mental_health_advice_tab": "Mental health tips"
   },
+  "maps": {
+    "error-header": "We are sorry",
+    "error-message": "At the moment, the desired data could not be obtained. Please try again later."
```

Please use underscore to continue the standard in the project and 
```suggestion
    "error_message": "At the moment, the desired data could not be obtained. Please try again later."
```